R/dm.R

```diff
@@ -481,6 +496,18 @@ print.dm <- function(x, ...) {
   invisible(x)
 }
 
+print_zoomed <- function(x) {
+  # so far only 1 table can be zoomed on
+  zoom <- cdm_get_zoomed_tbl(x)
+  tbl <- zoom %>% pull(zoom) %>% pluck(1)
+  name <- zoom %>% pull(table)
+  # FIXME: how do I get the text to be in grey colour (like above `tibble` print output)?
```

Comment:
Please implement `tbl_sum.dm()`, see `dplyr:::tbl_sum.grouped_df` .

Reply:
I implemented `tbl_sum.zoomed_df()`, see comment above the function for explanation.